### PR TITLE
FORT03.08 ASSET DRAINED FP UPDATE

### PR DESCRIPTION
**Pull request by 0xtaf**
Related issue: https://github.com/NethermindEth/Forta-Agents/issues/427

for liquidity removals, another alert type "ASSET-DRAINED-LIQUIDITY-REMOVAL" is added.

to detect liquidity removals, "Burn" event emission is used.

### CODE REVIEW

File: asset-drained/README.md [hunk 1]
```diff
@@ -44,6 +45,34 @@ This bot detects if a contract has had 99% or more of one of its assets drained
       - `confidence`: The confidence level of the address being a victim (0-1). Always set to `0.5`.
   - Addresses contain the list of addresses that received the assets from the drained contract
 
+- ASSET-DRAINED-LIQUIDITY-REMOVAL
+  - Fired when a contract has had 99% or more of one of its assets drained by _liqudity removal_
```
Comment (Vxatz) [inline asset-drained/README.md:49]:
```suggestion
  - Fired when a contract has had 99% or more of one of its assets drained by _liquidity removal_
```

File: asset-drained/package.json
Comment (Vxatz) [inline asset-drained/package.json:1]: Can you also add the `displayName` field? (requires upgrading to the latest sdk version)

File: asset-drained/src/agent.js [hunk 2]
```diff
@@ -18,10 +18,12 @@ const AddressType = require("./address-type");
 
 const ZERO = ethers.constants.Zero;
 const ERC20_TRANSFER_EVENT = "event Transfer(address indexed from, address indexed to, uint256 value)";
+const ERC20_BURN_EVENT = "event Burn(address indexed sender, uint amount0, uint amount1, address indexed to)";
```
Comment (Vxatz) [inline asset-drained/src/agent.js:21]: Should we add the Curve equivalent event?

Comment (0xtaf) [reply asset-drained/src/agent.js:21]: yes added 👍

File: asset-drained/package.json
Comment (0xtaf) [reply asset-drained/package.json:1]: done 👍

**Review by Vxatz — APPROVED** [review]

**Pull request merged**